gvr-360video/app/src/main/java/org/gearvrf/gvr360video/Minimal360Video.java

```diff
@@ -33,7 +33,7 @@
     /** Called when the activity is first created. */
     @Override
     public void onInit(GVRContext gvrContext) {
-        GVRScene scene = gvrContext.getNextMainScene();
+        GVRScene scene = gvrContext.getMainScene();
```

Comment:
With or without this change the model for apps with more than one GVRScene is not changed. If that has to happen, I rather have it as a separate pull request.

My thinking is to completely remove the capability to change or create a scene by end users. Changing the scene raises edge cases we deal with varying success to no clear benefit.

The real question is why would you need an app with 3 scenes. So far I have not seen a good reason to do so. gvr-immersivepedia does it but it can just as easily have its two scenes as regular scene objects in a gvrswitch node. Why support doing the same thing in more than one way?